CMakeLists.txt

```diff
@@ -185,6 +185,14 @@ if(ADDRESS_SANITIZER)
 endif()
 
 #------------------------------------------------------------------------------
+# Enable fingerid_test
+#------------------------------------------------------------------------------
+option(ENABLE_FINGERID_TEST "Enable test for fingerid (SiriusAdapter)" OFF)
+if(ENABLE_FINGERID_TEST)
```

Comment:
Did you try to just use the option throughout the CMake scripts?
I would guess that there is no need for a second variable.